src/services_models/GroupReserve.jl

```diff
@@ -0,0 +1,88 @@
+struct GroupReserve <: AbstractReservesFormulation end
+############################### Reserve Variables` #########################################
+"""
+This function checks if the variables for reserves were created
+"""
+function check_activeservice_variables(
+    psi_container::PSIContainer,
+    contributing_services::Vector{<:PSY.Service},
+)
+    for service in contributing_services
+        # Should pop an error if no such variable exists
+        reserve_variable =
+            get_variable(psi_container, PSY.get_name(service), typeof(service))
```

Comment:
No need to assign this to a variable if it's not used
```suggestion
        get_variable(psi_container, PSY.get_name(service), typeof(service))
```

Will this throw an informative exception if the variables don't exist?

Reply:
@daniel-thom did we implement a cleaner getter function that doesn't require calling `get_name` inside?

Reply:
@daniel-thom I believe it is `ActiveServiceVariable`? just changed to it. Please let me know if that's correct.

Reply:
I ended up not using get_variable in favor of a better error.